packages/components/app/styles/components/flyout.scss

```diff
@@ -50,58 +50,17 @@
   opacity: 0.5;
 }
 
-.hds-flyout__header {
-  display: flex;
-  flex: none;
-  gap: 16px;
-  align-items: flex-start;
-  padding: 16px 24px;
-  color: var(--token-color-foreground-strong);
-}
-
-.hds-flyout__icon {
-  flex: none;
-  align-self: center;
-}
-
-.hds-flyout__title {
-  flex-grow: 1;
-}
-
-.hds-flyout__tagline {
-  margin-bottom: 4px;
-}
-
-.hds-flyout__dismiss {
-  align-self: center;
-}
-
-.hds-flyout__description {
-  padding: 0 24px 16px;
-}
-
 .hds-flyout__body {
-  flex: 1 1 auto;
-  padding: 24px;
-  overflow-y: auto;
-  overscroll-behavior: contain;
   border-top: 1px solid var(--token-color-border-primary);
 }
 
 .hds-flyout__footer {
-  flex: none;
-  padding: 16px 24px;
-  background: var(--token-color-surface-faint);
   border-top: 1px solid var(--token-color-border-primary);
-
-  // Tertiary buttons must always be placed/aligned at the end of the row
```

Comment:
See comment above about this rule, which is specific to the `Flyout` only (so this rule should stay here)

Reply:
The exact same styles were originally in the modal.scss too though. See below.

```
  // Tertiary buttons must always be placed/aligned at the end of the row
  .hds-button-set {
    .hds-button--color-tertiary {
      margin-left: auto;
    }
  }
```